tests/test_vtk.py

```diff
@@ -45,6 +46,36 @@ def test_pluto_units(pluto_vtk_file):
         assert_allclose_units(getattr(ds, f"{u}_unit"), expected)
 
 
+def test_pluto_units_override(pluto_vtk_file):
+    file = pluto_vtk_file
+    ug = {
+        "time_unit": (2.0, "s"),
+        "length_unit": (4.0, "cm"),
+        "mass_unit": (5.0, "kg"),
+        "density_unit": (0.078125, "kg/cm**3"),
+        "velocity_unit": (2.0, "cm/s"),
+        "magnetic_unit": (62.66570686577499, "gauss"),
+    }
```

Comment:
Can you add a comment to explain how you chose these values (randomly ? or is this a specific set of consistent values ?)

Reply:
Sure. I just chose the first three values arbitraily and derived others to make them consistent.